InternshipProject#2/Controllers/WatcherController.cs

```diff
@@ -13,10 +15,12 @@ namespace InternshipProject_2.Controllers
     public class WatcherController : ControllerBase
     {
         private readonly IWatcherManager _manager;
```

Comment:
naming... naming.. naming... :)
It is very important to use proper naming. You might get it to this code after one year and you won't understand anything...

Reply:
Changed the name